.flake8

```diff
@@ -20,6 +20,8 @@ ignore =
     B950
     # No explicit stacklevel in warnings.warn. FIXME we should correct this in the code
     B028
+    # do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
+    E721
```

Comment:
Why ignore this?

Reply:
We use it all over the place and it's valid code using isinstance changes the behaviour of the code